### PR TITLE
escape JS comments inside js comments

### PR DESCRIPTION
**Pull request by FredKSchott**

### CODE REVIEW

File: src/document-util.ts [hunk 1]
```diff
@@ -426,12 +426,13 @@ export function attachCommentsToFirstStatement(
   const firstStatement: (estree.Statement|estree.ModuleDeclaration)&RecastNode =
       statements[0]!;
 
-  const recastComments: RecastComment[] = comments.map((c) => {
+  const recastComments: RecastComment[] = comments.map((comment) => {
+    const escapedComment = comment.replace('*/', '*\\/');
```
Comment (FredKSchott) [inline src/document-util.ts]: note that this only escapes the closing comment, leaving the opening comment untouched. This is syntactically all we need to fix the issue but for symmetry we may want to escape both. Open to thoughts here.

File: src/test/unit/project-converter_test.ts [hunk 2]
```diff
@@ -2485,6 +2485,42 @@ console.log('second script');
...
+  /**
+   *  Final comment
+   **/
+-->`
+      });
+
+      assertSources(await convert(), {
+        'test.js': `
+// comment in script
+/* /* First comment *\\/ */
```
Comment (stramel) [inline src/test/unit/project-converter_test.ts]: Seems odd that the first comment is now after the `// comment in script`

Comment (FredKSchott) [reply src/test/unit/project-converter_test.ts]: Agreed. This is common across all our tests, I'll add a separate issue to tackle

Comment (FredKSchott) [reply src/test/unit/project-converter_test.ts]: Agreed, filed as https://github.com/Polymer/polymer-modulizer/issues/281

File: src/document-util.ts (hunk 1 above)
Comment (justinfagnani) [reply src/document-util.ts]: +1

**Pull request merged**